tests/test_highly_variable_genes.py

```diff
@@ -692,10 +695,17 @@ def test_dask_consistency(adata: AnnData, flavor, batch_key, to_dask):
     adata_dask = adata.copy()
     adata_dask.X = to_dask(adata_dask.X)
 
-    output_mem, output_dask = (
-        sc.pp.highly_variable_genes(ad, flavor=flavor, n_top_genes=15, inplace=False)
-        for ad in [adata, adata_dask]
-    )
+    with (
+        pytest.warns(UserWarning, match="n_top_genes.*normalized dispersions")
```

Comment:
we already test for this in `test_cellranger_n_top_genes_warning` above, so maybe this test should be changed to not raise this warning?

Then I’d just make it visible here (Using the `always` warning filter action) and leave it like that for now.